docs/docs/troubleshooting/TROUBLESHOOTING.md

```diff
@@ -2,17 +2,37 @@
 
 This guide contains general tips on how to investigate an application deployment that doesn't work correctly.
 
-
 ## Cleanup Terraform state
 Terraform stores the lock and current state in the terraform backend file in `terraform-backend.tf`. 
 This file will be generated by the install process based on the content of `config.auto.tfvars`.
-If the user changes some key configuration items such as `region` or `environment_name` then you should 
-cleanup all temporary files generated by terraform and install modules again. 
+During the installation both installer and terraform generate some temporary files and all those files will be removed after uninstalling the environment.
+
+If the user has installed an environment and then changed any mandatory configuration items such as `region` or `environment_name` then they should 
+cleanup all temporary files generated by terraform before install the new environment. 
+
+To clean-up the local terraform files, run the following command:
+```
+./pkg/scripts/cleanup.sh -t
+```
+
+To clean-up the local generated variable files, run the following command:
+```
+./pkg/scripts/cleanup.sh -s
+```
 
-To cleanup the local terraform files, run the following command:
+You can combine both switches in one command to clean-up both generated terraform and variable files:
 ```
-./pkg/scripts/cleanup.sh
+./pkg/scripts/cleanup.sh -s -t
 ```
+!!! info "Temporary variable files will re-generate using the config file and override in install process."
+
+## Uninstall interruption
+### Symptom
+If you try uninstall using a different config file than what you installed the environment or using a different version of the project for uninstall you may face some problems in uninstall.
+Mostly terraform complains the resource cannot be removed because it is used. 
+
+### Solution
+In this case identify the resource and delete it manually using AWS console, then re-run uninstall. 
```

Comment:
Do they need to remove the state as well with `terraform state rm resource_address`?

Reply:
Does this will remove the actual resource in AWS side? I think this will just ignore the resource and allow terraform to continue the destroy while some resources will leave behind.